convert/convert.py

```diff
@@ -0,0 +1,33 @@
+from redbot.core import commands
+from redbot.core.utils.chat_formatting import pagify
+import discord
+from pint import UnitRegistry
+
+class Convert(commands.Cog):
+    """Convert related commands."""
+
+    def __init__(self):
+        self.__ureg = UnitRegistry()
+
+    @commands.command()
+    async def convert(self, ctx, *unit):
```

Comment:
Discord.py automatically uses the function args and the docstring to create the help message for each command. Could you either make the function args more descriptive _or_ include an example command in the docstring. You can see examples of this in some of the other cogs if you need styling hints :)

Reply:
I used `penis` as the template, as it was small enough. (Pun not intended)

As the input is not converted directly due to the intricate unit conversion and as I'm not completely comfortable on how commands and cogs should be made and I do not exactly know how you want your cogs, even by reading how you have done yours, I was unsure what to actually include in it.

I'll adjust with better documentation. Please tell me if I need to add more.